.kokoro/continuous/dependencies.cfg

```diff
@@ -0,0 +1,12 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+    key: "TRAMPOLINE_IMAGE"
+    value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+env_vars: {
+  key: "TRAMPOLINE_BUILD_FILE"
```

Comment:
we usually use two space indents, and this file is inconsistent

Reply:
Whitespace in the templates are being fixed upstream: https://github.com/googleapis/synthtool/pull/275